CMakeLists.txt

```diff
@@ -324,6 +325,10 @@ set(WABT_LIBRARY_SRC
 
 add_library(wabt STATIC ${WABT_LIBRARY_SRC})
 
+add_library(wasm-rt-impl STATIC wasm2c/wasm-rt-impl.c wasm2c/wasm-rt-impl.h)
+INSTALL(TARGETS wasm-rt-impl DESTINATION ${CMAKE_INSTALL_LIBDIR})
+INSTALL(FILES wasm2c/wasm-rt.h wasm2c/wasm-rt-impl.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
```

Comment:
The convention in this file is to use lowercase function names

Reply:
Adjusted.